### PR TITLE
On 1773 multiple answer questions

### PR DESCRIPTION
**Pull request by davidgouge**

### CODE REVIEW

File: src/SFA.DAS.ApplyService.Web/Views/Users/CreateAccount.cshtml [hunk 1]
```diff
@@ -61,8 +83,17 @@
...
-                    <input class="govuk-input" id="Email" name="Email" type="text" asp-for="Email"
-                        aria-describedby="email-hint" sfa-validation-for="FamilyName" sfa-validationerror-class="govuk-input--error" />
+                    <input
+                        class="govuk-input"
+                        id="Email"
+                        name="Email"
+                        type="text"
+                        asp-for="Email"
+                        aria-describedby="email-hint"
+                        sfa-validation-for="FamilyName"
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.ApplyService.Web/Views/Users/CreateAccount.cshtml]: This should Email - I know it was pre-existing code.

File: src/SFA.DAS.ApplyService.Web/Views/Users/CreateAccount.cshtml [hunk 2]
```diff
@@ -61,8 +83,17 @@
...
+                        sfa-validationerror-class="govuk-input--error"
+                        maxlength="320"
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.ApplyService.Web/Views/Users/CreateAccount.cshtml]: There doesn't seem to be anything specific in the ticket that suggests max length would be changed - or what the correct value should be (I'm guessing that this is the Email field in [Apply].[Contacts] - which is 256 or could eventually end up there so would be best to be the same max length)

File: src/SFA.DAS.ApplyService.Web/Views/Users/CreateAccount.cshtml [hunk 3]
```diff
@@ -47,7 +58,18 @@
...
+                        class="govuk-input govuk-!-width-two-thirds"
+                        id="FamilyName"
+                        name="FamilyName"
+                        type="text"
+                        asp-for="FamilyName"
+                        sfa-validation-for="FamilyName"
+                        sfa-validationerror-class="govuk-input--error"
+                        autocomplete="family-name"
+                        spellcheck="false"
+                        maxlength="10000"
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.ApplyService.Web/Views/Users/CreateAccount.cshtml]: Perhaps this should be the same length as most Family names in the Database e.g. 250

File: src/SFA.DAS.ApplyService.Web/Views/Users/CreateAccount.cshtml [hunk 4]
```diff
@@ -36,7 +36,18 @@
...
+                        class="govuk-input govuk-!-width-two-thirds"
+                        id="GivenName"
+                        name="GivenName"
+                        type="text"
+                        asp-for="GivenName"
+                        sfa-validation-for="GivenName"
+                        sfa-validationerror-class="govuk-input--error"
+                        autocomplete="given-name"
+                        spellcheck="false"
+                        maxlength="10000"
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.ApplyService.Web/Views/Users/CreateAccount.cshtml]: Perhaps this should be the same length as most Given names in the Database e.g. 250

File: src/SFA.DAS.ApplyService.Web/Views/Application/Standard/FindStandard.cshtml [hunk 5]
```diff
@@ -40,7 +40,7 @@
                         <span id="search-string-error" class="govuk-error-message">Enter 2 or more characters to search</span>
                     }
 
-                    <input class="govuk-input govuk-input--width-20 ui-autocomplete-input @(!ViewData.ModelState.IsValid || showErrors ? "govuk-input--error" : "")" id="standard-name" name="StandardToFind" asp-for="@Model.StandardToFind"type="text" value="" autocomplete="off" maxlength="500">
+                    <input class="govuk-input govuk-input--width-20 ui-autocomplete-input @(!ViewData.ModelState.IsValid || showErrors ? "govuk-input--error" : "")" id="standard-name" name="StandardToFind" asp-for="@Model.StandardToFind"type="text" value="" autocomplete="off" maxlength="10000">
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.ApplyService.Web/Views/Application/Standard/FindStandard.cshtml]: Why is this increase by so much - is this the same as StandardCollation (Title) which is 500 character is the Assessor database.

File: src/SFA.DAS.ApplyService.Web/Views/Application/Standard/FindStandard.cshtml [hunk 6]
```diff
@@ -40,7 +40,7 @@
                         <span id="search-string-error" class="govuk-error-message">Enter 2 or more characters to search</span>
                     }
 
-                    <input class="govuk-input govuk-input--width-20 ui-autocomplete-input @(!ViewData.ModelState.IsValid || showErrors ? "govuk-input--error" : "")" id="standard-name" name="StandardToFind" asp-for="@Model.StandardToFind"type="text" value="" autocomplete="off">
+                    <input class="govuk-input govuk-input--width-20 ui-autocomplete-input @(!ViewData.ModelState.IsValid || showErrors ? "govuk-input--error" : "")" id="standard-name" name="StandardToFind" asp-for="@Model.StandardToFind"type="text" value="" autocomplete="off" maxlength="500">
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.ApplyService.Web/Views/Application/Standard/FindStandard.cshtml]: In a subsequent change a similar field is increase from 500 to 10000 - this is given a new 500 limit though - inconsistent?

Comment (gdbarnes) [reply src/SFA.DAS.ApplyService.Web/Views/Application/Standard/FindStandard.cshtml]: This is the same file, so is outdated.

File: src/SFA.DAS.ApplyService.Web/Views/Users/CreateAccount.cshtml (hunk 1 above)
Comment (ChrisJWoodcock) [reply src/SFA.DAS.ApplyService.Web/Views/Users/CreateAccount.cshtml]:
```suggestion
                        sfa-validation-for="Email"
```

File: src/SFA.DAS.ApplyService.Web/Views/Users/CreateAccount.cshtml (hunk 4 above)
Comment (ChrisJWoodcock) [reply src/SFA.DAS.ApplyService.Web/Views/Users/CreateAccount.cshtml]: The reason for increase was to match 10000 as that is the character limit in the WAF;

**Pull request merged**